filebeat/.gitignore

```diff
@@ -3,10 +3,12 @@
 /docs/html_docs
 
 filebeat
+filebeat.exe
 
 build
 _meta/kibana
 _meta/module.generated
 _meta/beat.yml
 _meta/beat.reference.yml
 /tests/load/logs
+.vscode
```

Comment:
Try not to modify this to much, we'd like to be able to PR this back upstream if possible